.github/workflows/terraform-apply.yaml

```diff
@@ -46,7 +46,22 @@ jobs:
         working-directory: ${{ inputs.module-directory }}
         run: |
           terraform init
-          terraform apply -var "channel=${{ inputs.channel }}" -var "model_name=${{ inputs.model }}" --auto-approve
+
+          ARGS=""
+
+          if [ -n "${{ inputs.channel }}" ]
+          then
+            ARGS="${ARGS} -var channel=${{ inputs.channel }}"
+          fi
+
+          if [ -n "${{ inputs.model }}" ]
+          then
+            ARGS="${ARGS} -var model_name=${{ inputs.model }}"
+          fi
```

Comment:
Since `model` has a default value, we can skip this check and just add it by default in line 50 with sth like
```
ARGS="-var model_name=${{ inputs.model }}"
```